sormas-e2e-tests/src/main/java/org/sormas/e2etests/services/ContactService.java

```diff
@@ -151,9 +147,30 @@ public Exposure buildGeneratedExposureDataForContact() {
         .contactToBodyFluids(YesNoUnknownOptions.NO)
         .handlingSamples(YesNoUnknownOptions.NO)
         .typeOfPlace(TypeOfPlace.HOME)
-        .continent("Africa")
```

Comment:
but this would cause failures into other tests. If you really need this change for your test, create a new entity service builder to cover this
like: buildGeneratedExposureDataForContact(String continent, ......) so we can also reuse it in other situations